classes/index.js

```diff
@@ -1,15 +1,15 @@
 'use strict';
```

Comment:
Nit: there's a few places where `use strict` can be removed

Reply:
Done!